### ISSUE TITLE
Accessibility notes for each component

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#3111]
https://vuetifyjs.com/en/components/alerts/#accessibility

Should be done after #3110 (could be done in parralel for each component)

**Comment by kushich** [comment]
moved to https://github.com/epicmaxco/vuestic-ui/issues/3223
